x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -155,37 +159,61 @@ export function RuleDetailsPage() {
           ? [
-              <>
-                <EuiPopover
-                  id="contextRuleEditMenu"
-                  isOpen={isRuleEditPopoverOpen}
-                  closePopover={() => setIsRuleEditPopoverOpen(false)}
-                  button={
-                    <EuiButtonIcon
-                      display="base"
-                      size="m"
-                      iconType="boxesHorizontal"
-                      aria-label="More"
-                      onClick={() => setIsRuleEditPopoverOpen(true)}
-                    />
-                  }
-                >
-                  <EuiButtonEmpty
-                    size="m"
-                    iconType="pencil"
-                    onClick={() => {
-                      setIsRuleEditPopoverOpen(false);
-                      setEditFlyoutVisible(true);
-                    }}
+              <EuiFlexGroup direction="rowReverse" alignItems="center">
+                <EuiFlexItem>
+                  <EuiPopover
+                    id="contextRuleEditMenu"
+                    isOpen={isRuleEditPopoverOpen}
+                    closePopover={() => setIsRuleEditPopoverOpen(false)}
+                    button={
+                      <EuiButtonIcon
+                        display="base"
+                        size="m"
+                        iconType="boxesHorizontal"
+                        aria-label="More"
+                        onClick={() => setIsRuleEditPopoverOpen(true)}
+                      />
+                    }
                   >
-                    <EuiText size="m">
-                      {i18n.translate('xpack.observability.ruleDetails.editRule', {
-                        defaultMessage: 'Edit rule',
+                    <EuiButtonEmpty
+                      size="m"
+                      iconType="pencil"
+                      onClick={() => {
+                        setIsRuleEditPopoverOpen(false);
+                        setEditFlyoutVisible(true);
+                      }}
+                    >
+                      <EuiText size="m">
+                        {i18n.translate('xpack.observability.ruleDetails.editRule', {
+                          defaultMessage: 'Edit rule',
+                        })}
+                      </EuiText>
+                    </EuiButtonEmpty>
+                  </EuiPopover>
+                </EuiFlexItem>
+                <EuiSpacer size="s" />
+                <EuiFlexItem>
+                  <EuiTitle size="xxs">
+                    <EuiFlexItem>
+                      {i18n.translate('xpack.observability.ruleDetails.triggreAction.status', {
+                        defaultMessage: 'Status',
                       })}
-                    </EuiText>
-                  </EuiButtonEmpty>
-                </EuiPopover>
-              </>,
+                    </EuiFlexItem>
+                  </EuiTitle>
+
+                  {triggersActionsUi.getRuleStatusDropdown({
```

Comment:
@fkanout You need this PR to be merged https://github.com/elastic/kibana/pull/131108 first, otherwise you will have some failing tests. My PR fixes this https://github.com/elastic/kibana/pull/131108/commits/193dc513a20e78775c4f7c71bf9d3172a6b1a4ca